sequencer/src/header.rs

```diff
@@ -96,6 +97,9 @@ impl Committable for Header {
         self.fee_merkle_tree_root
             .serialize_with_mode(&mut fmt_bytes, ark_serialize::Compress::Yes)
             .unwrap();
+        let mut fee_info_bytes = vec![];
+        self.fee_info
+            .serialize_with_mode(&mut fee_info_bytes, ark_serialize::Compress::Yes);
```

Comment:
Hmm I think this serialization makes the commitment needlessly opaque (and difficult to port to other languages). Better to implement `Committable for FeeInfo` and then we can just use `.field("fee_info", self.fee_info.commit())`